custom/icds_reports/migrations/0157_updated_unique_constraint.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Verified that its working:

Below is the image of table before and after migration

<img width="1409" alt="Screenshot 2019-12-25 at 12 32 50 AM" src="https://user-images.githubusercontent.com/21310920/71423265-2ed38300-26ae-11ea-8b4e-4212fe18d77a.png">